### ISSUE TITLE
Database functions

### ISSUE THREAD

**Issue by aaort** [supabase/supabase#12438]
# Bug report

## Can't view my database functions, getting client side error.

I'm new to Supabase and I was trying to create database function which would get triggered every time a new user row is inserted into `auth.users` table (sign up basically). I'm using Next.js & Supabase and after following [this guide](https://egghead.io/lessons/supabase-use-supabase-to-subscribe-to-database-events-with-postgres-triggers) and [this code sample](https://database.dev/handle-new-user) I tried to implement my own function and after several attempts for some reason now I can't even visit my database functions page.

## To Reproduce

I can't tell for sure how to reproduce this issue because I can't tell which steps caused it exactly.
It happened while I was trying to create database function.

## Expected behavior

I would expect to be able to view/manage database functions from the functions tab of my project's database view.

## Screenshots

What I see when trying to visit `database/functions`  page.
<img width="753" alt="Screen Shot 2023-02-15 at 15 08 10" src="https://user-images.githubusercontent.com/115251394/219011203-c92a1b70-a408-484c-884b-c6029198b0e6.png">
Error.
<img width="887" alt="Screen Shot 2023-02-15 at 14 54 56" src="https://user-images.githubusercontent.com/115251394/219008324-14ff57fd-3522-429b-a6e9-5f2b21c39846.png">
Additionally, warnings that also appear in console and might be related to this issue
<img width="1440" alt="Screen Shot 2023-02-15 at 15 08 53" src="https://user-images.githubusercontent.com/115251394/219011375-5dd324a1-e884-4bf7-af53-d3c04130dbd7.png">

## System information

- OS: macOS
- Browser: Chrome

## Additional context

Thank you to everyone working on this product, I very much like using it and would love to contribute myself by fixing this issue at least, but I don't have enough knowledge. However I'm willing to collaborate in any possible way around this issue, if provided information wasn't enough.

**Comment by GaryAustin1** [comment]
There does not appear to be a general issue at least on Windows (Chrome/Firefox), which is all I have to test with.  No other reports of this.
You might hard refresh your browser and restart your instance from dashboard general settings.

I skimmed the guides you linked and not sure how anything done wrong in them would lead to the errors you are seeing.

Either more info will be needed, start a new instance, or contact support if it is something wrong on the instance side.  This is a Supabase hosted instance?

**Comment by MildTomato** [comment]
Long shot but, maybe a function got deployed without a name.
Just a guess judging by the toLowerCase() type error

**Comment by MildTomato** [comment]
@aaort could you open up a support ticket on https://app.supabase.com/support/new ?
This is possibly an isolated issue to your instance.

**Comment by MildTomato** [comment]
I'll leave this ticket open as, regardless, dashboard pages should fail gracefully and not make pages inaccessible

**Comment by aaort** [comment]
I tried your suggestions but unfortunately with no success. The rest of project services work just fine though.
Yes, it is a Supabase hosted project.

**Comment by aaort** [comment]
Eventually I have created another Supabase project to get rid of this error.
Does someone have time and willing to work with this or should I close this issue ?

**Comment by kiwicopple** [comment]
I'll flag this to the Dashboard team, but I think you can close this one for now @aaort 👍 

Thanks for reporting it!

**Comment by JakeRoyRandall** [comment]
I have a similar issue with in that I cannot get database functions to show up in the Supabase UI, though I can see they exist because I can attach them to triggers. This problem has persisted for at least 6 hours, through multiple instance restarts & logging in and out. I've attached a short clip. Apologies if I'm missing something.

https://user-images.githubusercontent.com/57545729/219542162-857e1eaa-a6d8-4409-9962-c71388fec7a9.mp4

**Comment by joshenlim** [comment]
@aaort just to check - do you remember what was the function that you last created which resulted in this problem? I'm trying to reproduce the issue

**Comment by aaort** [comment]
@joshenlim Sure, I named it handle_new_user() and created it using the database/functions client interface. I deleted it later and I believe that this action was the reason behind all this.

After that I tried to look up that function using SQL editor (since I couldn't open the appropriate page for that) but I was unable to do that, meaning that the function was actually deleted and I didn't have any other functions defined. 

Question arise, if there were no functions defined than name of which function client was trying to call `toLowerCase()` on.
